src/components/LocalizableTextInput/formData.ts

```diff
@@ -4,14 +4,21 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-type LocalizableTextFormData =
-  | {
-      localized: true;
-      stringId: string;
-    }
-  | {
-      localized: false;
-      text: string;
-    };
+export interface RichTextProperties {
+  zap?: true;
```

Comment:
Curious why zap is true here, instead of boolean - is it just exposing the zap property in the JSON?
Not super important since we don't have any current messages that need a zap, just curious about the handling